File: basic/src/main/java/com/yuanshenbin/basic/imgloader/IImageLoaderProxy.java
package com.yuanshenbin.basic.imgloader;

import android.view.View;

/**
 * author : yuanshenbin
 * time   : 2021-09-24
 * desc   :
 */
public interface IImageLoaderProxy {

    void displayImage(Object url, View view);

    void placeholder(int resId);

    void options(ImageOptions l);
}

File: basic-kotlin/src/main/java/com/yuanshenbin/basic/imgloader/IImageLoaderProxy.kt
package com.yuanshenbin.basic.imgloader

import android.view.View

/**
 * author : yuanshenbin
 * time   : 2021-09-24
 * desc   :
 */
interface IImageLoaderProxy {
    fun displayImage(url: Any?, view: View?)
    fun placeholder(resId: Int)
    fun options(l: ImageOptions?)
    val cacheSize: Long
    fun clearCache()
}